llvm/include/llvm/ProfileData/Coverage/CoverageMapping.h

```diff
@@ -380,7 +380,70 @@ struct MCDCRecord {
   /// are effectively ignored.
   enum CondState { MCDC_DontCare = -1, MCDC_False = 0, MCDC_True = 1 };
 
-  using TestVector = llvm::SmallVector<CondState>;
+  /// Emulate SmallVector<CondState> with a pair of BitVector.
+  ///
+  ///          True  False DontCare (Impossible)
+  /// Values:  True  False False    True
+  /// Visited: True  True  False    False
+  class TestVector {
+    BitVector Values;  /// True/False (False when DontCare)
+    BitVector Visited; /// ~DontCare
+
+  public:
+    /// Assume filling DontCare.
```

Comment:
Why do you need to pass Cond when it is not being used?

Reply:
It's because the only user (`TV` in `findExecutedTestVectors()`) specifies `DontCare` explicitly. I wanted to leave it since I thought it was more descriptive.

I can prune the initial value in this.